source/js/addons.js

```diff
@@ -0,0 +1,183 @@
+function getAddonList() {
+    $.getJSON(apiUrl, function (data) {
+        addons = data.data.sort(sortByName);
+        resortAddons(data.data);
+        rerenderAddons();
+    });
+}
+
+function resortAddons(data) {
+    addonsGameLibrary = [];
+    addonsMetadataProvider = [];
+    addonsGeneric = [];
+    addonsThemeDesktop = [];
+    addonsThemeFullscreen = [];
+    $.each(data, function (_key, val) {
+        switch (val.type) {
+            case 1:
+                addonsMetadataProvider.push(val);
+                break;
+            case 2:
+                addonsGeneric.push(val);
+                break;
+            case 3:
+                addonsThemeDesktop.push(val);
+                break;
+            case 4:
+                addonsThemeFullscreen.push(val);
+                break;
+            case undefined:
+                addonsGameLibrary.push(val);
+                break;
+            default:
+        }
+    });
+}
+
+function rerenderAddons() {
+    renderAddonsSection(addonsMetadataProvider, "addonsMetadataProvider", "Metadata Sources");
+    renderAddonsSection(addonsGeneric, "addonsGeneric", "Generic");
+    renderAddonsSection(addonsThemeDesktop, "addonsThemeDesktop", "Themes Desktop");
+    renderAddonsSection(addonsThemeFullscreen, "addonsThemeFullscreen", "Themes Fullscreen");
+    renderAddonsSection(addonsGameLibrary, "addonsGameLibrary", "Libraries");
+}
+
+function sanitize(input) {
+    var output = input.replace(/<script[^>]*?>.*?<\/script>/gi, '').
+        replace(/<[\/\!]*?[^<>]*?>/gi, '').
+        replace(/<style[^>]*?>.*?<\/style>/gi, '').
+        replace(/<![\s\S]*?--[ \t\n\r]*>/gi, '').
+        replace(/&nbsp;/g, '');
+    return output;
+}
+
+function renderAddonsSection(data, type, name) {
+    var htmlItems = [];
+    if (data.length > 0) {
+        $.each(data, function (_key, val) {
+            htmlItems.push(" \
+            <li class='list-group-item'> \
+                <div class='row'> \
+                    <div class='col-sm-2'> \
+                        " + (val.iconUrl != undefined ? "<img src=" + encodeURI(val.iconUrl) + " width=120 loading='lazy'></img>" : "") + "\
+                    </div> \
+                    <div class='col-sm-9'> \
+                        <h4>" + sanitize(val.name) + "</h4> \
+                        <p class='addon-description'>" + sanitize(val.description == undefined ? val.shortDescription : val.description) + "</p> \
+                " + (val.screenshots != undefined ? "<p> \
+                " + $.map(val.screenshots,
+                function (key, _val) {
+                    return '<a href="' + encodeURI(key.image) + '" target="_blank"><img class="addon-screenshot" src="' + encodeURI(key.thumbnail) + '" loading="lazy"></img></a>';
+                }).join('') + "</p>" : '') + " \
+                <p> \
+                <a href=" + encodeURI(val.sourceUrl) + " target='_blank'>Source code</a> \
+                " + (val.links != undefined ? " | \
```

Comment:
Avoid unquoted attributes with `encodeURI()`:
```suggestion
                }).join('') + "</p>" : '') + ' \
                <p> \
                <a href="' + encodeURI(val.sourceUrl) + "' target="_blank">Source code</a> \
                ' + (val.links != undefined ? " | \
```